apps/angular/src/app/features/auth/login/login.component.ts

```diff
@@ -0,0 +1,72 @@
+import { ChangeDetectionStrategy, Component, DestroyRef, inject } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { FormControl, FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { AuthService } from '@js-camp/angular/core/services/auth.service';
+import { catchApiError } from '@js-camp/angular/core/utils/rxjs/catch-api-error';
+import { ApiError } from '@js-camp/core/models/api-error';
+import { AuthenticationConstants } from '@js-camp/core/utils/authentication-constants';
+import { of, tap } from 'rxjs';
+
+/** Login form controls. */
+interface LoginFormControls {
+
+	/** Email control. */
+	readonly email: FormControl<string>;
+
+	/** Password control. */
+	readonly password: FormControl<string>;
+}
+
+/** Login component. */
+@Component({
+	selector: 'login',
```

Comment:
Don't forget to add app prefix to all comonent/directive/pipe selectors in your app.
If I'm not mistaken Angular CLI does this automatically but maybe something is configured incorrectly in the camp boilerplate (or in you project).
```suggestion
	selector: 'camp-login',
```

Reply:
Please, check other components as well. App prefix is missing there as well.